Cargo.toml

```diff
@@ -11,20 +11,20 @@ edition = "2018"
 crate-type = ["cdylib"]
```

Comment:
We should bump the version up above on line 3 to `0.5.6` as part of this PR. Also, as you mentioned in the PR description, add a new `CHANGELOG.md` entry.

Reply:
Done